### ISSUE TITLE
A simpler way for images and captions

### ISSUE THREAD

**Issue by mml1** [babyboomer53/TeaCozy#3]
The beauty about programming is that there are many ways to do the same thing with some ways being more efficient than others. I'm glad you spent time exploring different ways to have an image with a caption; curiosity is very important trait to have as a developer.  

In this case, the more efficient way to have an image with a caption is to hold both image and caption in a div. With in that div, give the image a height and make both image and caption block elements, display:block;, which will cause both image an caption to take up the whole width of the div thus making them be on top of each other. Be sure to have a div containing all the items too.  

Something like:
**HTML**
```
        <div class="item">
          <img src="./resources/images/img-berryblitz.jpg" alt="Fall Berry Blitz Tea" />
          <span>Fall Berry Blitz Tea</span>
        </div>
```
**CSS**
```
.item img {
  height: 200px;
  margin: 10px;
  display: block;
}

.item span {
  display: block;
  text-align: center;
}

```
https://github.com/babyboomer53/TeaCozy/blob/master/index.html#L47-L79

**Comment by babyboomer53** [comment]
When coding the "featured tea" section in the initial release of this document, I failed to implement that solution using flex-boxes. Instead, I used table elements. My research revealed that the use of tables for this purpose is not the best practice. My experience was that tables are relatively unwieldy. Intent on remediating that, I opened a new branch (flex-remediation) for this repository. The flex-remediation branch contains a version of this project in which the tables have been replaced by flex-boxes.

**Comment by babyboomer53** [comment]
I opened a new branch (flex-remediation) for this repository, which includes a version of the project that replaces the tables with flex boxes. Reviewing the video tutorial on flex boxes was very helpful. I suspect I'll do that a couple more times before I'm completely comfortable with the concept.

I have tested the updated version, and will be merging the branches shortly. in addition to  its primary objective, this project is giving me an opportunity to get more practice with git.

Thank you for your feedback!
